web/packages/plugins/shared/common/src/components/TreeView.tsx

```diff
@@ -149,7 +150,12 @@ export const TreeView = (props: {
         if (NodeWrapper) {
           return (
             // @ts-ignore
-            <NodeWrapper node={node} key={node.nodeId} removeNode={removeNode}>
+            <NodeWrapper
+              node={node}
+              key={node.nodeId}
+              onSelect={NodeWrapperOnClick}
+              removeNode={removeNode}
```

Comment:
does not look like the NodeWrapper component used in DestiantionPicker (SelectPackageButton) has a prop called removeNode.

typescript would notify us about errors like this, so it would be best to remove
the //@ts-ignore on line 152.

to do this:
- update the TreeView component prop type to: 
NodeWrapper?: React.FunctionComponent<NodeWrapperProps>
- in TreeView.tsx prop type NodeWrapperProps and use this as the prop type in SelectPackageButton component in DestiantionPicker.tsx

Reply:
done